cmd/spire-server/cli/entry/create.go

```diff
@@ -45,6 +45,9 @@ type CreateConfig struct {
 
 	// Expiry of entry
 	EntryExpiry int64
+
+	// DNS entries for SVIDs based on this entry
+	DNSList StringsFlag
```

Comment:
DNSNames feels clearer, more aligned with `Selectors`, and I think will work better in datastore layer too